src/quick-start.md

```diff
@@ -2,45 +2,59 @@
 
 The Kubewarden stack is made of the following components:
 
-* An arbitrary number of `ClusterAdmissionPolicy` resources: this is how policies
-  are defined inside of Kubernetes
-* An arbitrary number of `PolicyServer` resources, which represent a Deployment of a Kubewarden `policy-server`.
-  This component loads all the policies defined by the administrators and evaluates them
-* A Deployment of `kubewarden-controller`: this is the controller
-  that monitors the `ClusterAdmissionPolicy` resources and interacts
-  with the Kubewarden `policy-server`
+* An arbitrary number of `ClusterAdmissionPolicy` resources: this is how policies are defined inside Kubernetes
+* An arbitrary number of `PolicyServer` resources: this component represents a Deployment of a Kubewarden `policy-server`. The policies defined by the administrators are loaded and evaluated by the Kubewarden `policy-server`
+* A Deployment of `kubewarden-controller`: this is the controller that monitors the `ClusterAdmissionPolicy` resources and interacts with the Kubewarden `policy-server` components
+
+## Installation
+
+> **PREREQUISITES:**
+>
+> Currently, the chart depends on cert-manager. Make sure you have [`cert-manager` installed](https://cert-manager.io/docs/installation/) *before* installing the kubewarden-controller chart.
+>
+> You can install the latest version of `cert-manager` by running the following commands: 
+>
+```shell
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml
 
-## Install
+kubectl wait --for=condition=Available deployment --timeout=2m -n cert-manager --all
+```
 
-The Kubewarden stack can be deployed using a helm chart.
-Currently, the chart depends on cert-manager. Make sure you have [`cert-manager` installed](https://cert-manager.io/docs/installation/) and then install the kubewarden-controller chart. For example:
+The Kubewarden stack can be deployed using a `helm` charts as follow:
 
-```console
+```shell
 helm repo add kubewarden https://charts.kubewarden.io
```

Comment:
Nit: we were missing a `helm repo update` afterwards, before this PR.

Reply:
Before I wrote the quickstart, I went through it "as is", and adding the `helm` repo and then installing Kubewarden didn't throw any error.

Could it be a `helm` cli version maybe? I did all my testing with the version 3.6.3 which comes with Rancher Desktop

Reply:
maybe it was an old version thing! tried here, I don't need to do helm repo update either.